met-web/src/components/engagement/old-view/widgets/WhoIsListeningWidget.tsx

```diff
@@ -102,56 +198,75 @@ const WhoIsListeningWidget = ({ widget }: WhoIsListeningWidgetProps) => {
                             lg={12}
                             xl={9}
                         >
-                            <Grid item container justifyContent={{ xs: 'center', md: 'flex-start' }} xs={12}>
-                                <BodyText size="large" bold>
+                            <Grid justifyContent={{ xs: 'center', md: 'flex-start' }} item xs={12}>
+                                <BodyText sx={contactNameStyles} aria-label="Contact name:">
                                     {contact.name}
                                 </BodyText>
                             </Grid>
                             <When condition={Boolean(contact.title)}>
-                                <Grid container justifyContent={{ xs: 'center', md: 'flex-start' }} item xs={12}>
-                                    <BodyText>{contact.title}</BodyText>
+                                <Grid justifyContent={{ xs: 'center', md: 'flex-start' }} xs={12}>
+                                    <BodyText sx={contactTitleStyles} aria-label="Contact job title:">
```

Comment:
Hmm, I'm not necessarily against this but just curious on why you added the aria label? Is it clear to sighted users that this is a job title, but not to non-sighted users?

Reply:
I guess it depends how you think it should be read. To me, it should be like name: John Smith, contact job title: minister of mining, etc. But it could also just read the fields, like John Smith, Minister of Mining.

Reply:
I changed it to the shorter way.